Code/GraphMol/Fingerprints/testFingerprintGenerators.cpp

```diff
@@ -2423,6 +2423,46 @@ void testBulkFP() {
   BOOST_LOG(rdErrorLog) << "  done" << std::endl;
 }
 
+void testAtomPairFPDifference() {
```

Comment:
it's fine to add tests to the old-school testing code, but it is simpler if you add catch tests instead.